app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+    - Database schemas are largely reusable
+    - Public APIs work seamlessly for Normandy and Nimbus experiments
+    - External service integrations (bugzilla/remote settings/email/auth0) work seamlessly
+  - Comprehensive development/testing/deployment infrastructure in place
+    - 100% unit testing coverage across Python/JS/TS
+    - Automated Selenium Integration tests for frontend
+    - Continuous Deployment
+- Cons
+  - Existing workflow/UI not applicable
+  - Some database schemas require refactoring to isolate Normandy and Nimbus concerns
+  - Careful consideration that adding Nimbus features does not disturb existing functionality
+  - Lack of static typing (mypy can be added with effort)
+  - Supporting both Normandy and Nimbus over time could cause unwanted complexity/duplication
+  - [Current Experimenter codebase may not align with FXA tools/processes/practices](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+  - Somewhat constricted by core designs decisions that were made in service of existing Normandy experimentation workflows that may not apply to Nimbus
+
+### Start A New Application In Python/Django
+
+Creating a new web application in Python/Django would allow us to reuse much of the existing code/infrastructure where applicable while clearly isolating the concerns of supporting existing experiment types and building out new Nimbus functionality.
+
+- Pros
+  - Start by cloning the repo and deleting all the unnecessary UI/DB/APIs
+  - Get a big head start by not needing to reimplement the reusable parts
+  - Isolate Nimbus concerns without fear of breaking Normandy experiments
+  - Consistent development and deployment workflow for both old and new applications
+  - Opportunity to rethink core code/architectural/tooling decisions where appropriate
+- Cons
+  - Lose single source of truth
+    - Possibly require people to know which application to refer to depending on experiment type
+    - Possibly create an API proxy in front of both to have a single endpoint for dependent callers
+    - More difficult to correlate experiments/learnings across the two applications
+  - Changes that must apply to both Normandy and Nimbus experiments must be made in two places while they're both being actively maintained
+  - Divergence in tooling/processes over time may create confusion when context switching between the two
+  - Possibly still somewhat constricted by past core design decisions
+  - [Python/Django may not align with FXA tools/processes](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+
+### Start A New Application in TypeScript/Node
+
+Creating a new web application in TypeScript/Node would allow us to completely restart our thinking about the entire structure of the application which would result in both code and workflow/UI that are purpose built from the ground up to solely address the concerns of Nimbus experiments.
+
+- Pros
```

Comment:
The assisting FxA team has a strong background in TypeScript/Node as well as the supporting libraries, frameworks, and processes around that.  I think that's a definite `Pro` here*, particularly since we'd like to move quickly on this.

* I'm wondering if I would call it a `Con` for the others....  We have Python/Django experience, but it's years old.  Since rapid development and `Fitting cohesively into the FXA engineering pipeline/culture` are decision drivers listed above, I guess I would.

Reply:
Yep that's exactly what I meant with that point 👍

Reply:
I'm repeating myself here, but `Fitting cohesively into the FXA engineering pipeline/culture` is more relevant if the FxA team has long term maintenance / ownership of this app. In the equally likely case that the Data team has long term ownership, then this decision drive should be replaced (or at least tempered) with `Supporting smooth transition into the Data Tools engineering pipeline/culture`, which is strongly Python focused.